### ISSUE TITLE
Random delimiter(oddělovač) in CSV export

### ISSUE THREAD

**Issue by BalckhatinRED** [ondra-novak/mmbot#9]
Picture more than thousands words :)  ... marginal priority
![CSV_export](https://user-images.githubusercontent.com/57764583/68872967-22e3c100-06ff-11ea-8f38-175c391d8b0e.jpg)

**Comment by ondra-novak** [comment]
I don't follow. Where is the problem?

**Comment by BalckhatinRED** [comment]
If you use function "Text to columns" in Excel is very hard to delimit each
values if the delimiter is different in each value.

Good:
Hodnota1, Hodnota2, Hodnota3, Hodnota4, Hodnota5
Hodnota1; Hodnota2; Hodnota3; Hodnota4; Hodnota5
Hodnota1.Hodnota2.Hodnota3.Hodnota4.Hodnota5

Wrong:
Hodnota1, Hodnota2; Hodnota3.Hodnota4#Hodnota5

As I mention. just detail, but in case you want to analyze data in excel is
hard to put csc in to columns.

**Comment by ondra-novak** [comment]
All fields are delimited by comma (always) and strings are in quotes. It seems that your file has been corrupted somewhere. There can be also some very rare specific issues for particular browser, because the CVS file is built by the javascript code

https://github.com/ondra-novak/mmbot/blob/master/www/res/code.js#L920
(you can see, that each row is created by JSON.stringify, so it is actually JSON array without the first and last brace.)

![image](https://user-images.githubusercontent.com/9439234/70348719-93e04980-1863-11ea-91f2-5536764764d3.png)

**Comment by BalckhatinRED** [comment]
I played more with LibreOffice text importer.(Vivaldi as a browser) You are right it's ok. I think problem will be in my hands(between a keyboard and the chair).

I used q.mark+comma+q.mark as a delimiter, as in the first line those 3 chars seems like a delimiter. But in data lines bellow the delimiter is not those 3 characters. The quotation mark is in some columns only one or none, and sometimes on left, sometime son right side of comma. 

Long story short, now it somehow works even I use only comma, if I use all 3 chars. I updated LibreOffice, I had some old version. Maybe it help. Anyway. It works now :) Sorry
